website/docs/deployment.mdx

```diff
@@ -200,7 +200,7 @@ If you decide to use the `website` folder as Netlify's base directory, Netlify w
 
 By default, Netlify adds trailing slashes to Docusaurus URLs.
 
-It is recommended to disable the Netlify setting `Post Processing > Asset Optimization > Pretty Urls` to prevent lowercased URLs, unnecessary redirects, and 404 errors.
+It is recommended to disable the Netlify setting `Build & Depoly > Post Processing > Pretty Urls` to prevent lowercased URLs, unnecessary redirects, and 404 errors.
```

Comment:
```suggestion
It is recommended to disable the Netlify setting `Build & Deploy > Post Processing > Pretty URLs` to prevent lowercased URLs, unnecessary redirects, and 404 errors.
```